pepagent/pepagent.py

```diff
@@ -68,19 +70,25 @@ def upload_project(
         project: peppy.Project,
         namespace: str = None,
         name: str = None,
+        tag: str = None,
         anno: dict = None,
+        update: bool = False,
     ) -> None:
         """
         Upload project to the database
```

Comment:
Should we split `upload_project` and `update_project`? Having one is convenient, but it might get confusing. That way the logic inside can be much simpler too since we aren't juggling so many edge cases.

Reply:
ok, it make sense. I will change it!